### ISSUE TITLE
Add filter statistics

### ISSUE THREAD

**Issue by mikropsoft** [AdguardTeam/AdGuardHome#7023]
### Prerequisites

- [X] I have checked the [Wiki](https://github.com/AdguardTeam/AdGuardHome/wiki) and [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions) and found no answer

- [X] I have searched other issues and found no duplicates

- [X] I want to request a feature or enhancement and not ask a question

### Feature request

As with Nextdns, it would be great to be able to see statistics on how many domains are blocked by which blacklist.

**Comment by ainar-g** [comment]
Duplicate of #4524
